sources/platform/management/subscription/whitelist-nodes.md

```diff
@@ -0,0 +1,40 @@
+
+# Whitelisting Build Nodes
+Knowing the IP addresses of the build machines Shippable uses can be helpful when you need them to whitelist specific IP ranges so that build nodes can freely talk to machines/tools within their VPC or network.
+
+## Enabling Whitelisting of Build Nodes
+You can enable whitelisting of build nodes in your subscription by creating a node pool with NAT enabled or enabling NAT in existing node pools.
+All the nodes that come up in the NAT enabled node pool will have static IP. Only Paid subscriptions will be able to use this feature.
+
+### Creating a NAT enabled node pool
+You can create a new NAT enabled node pool by clicking on the `+` icon on top right corner of the Node Pools page. This will open the **ADD NODE POOL** page.
+<img src="/images/platform/management/add_nat_node_pool.png" alt="Subscription NAT Node Pools add for Shippable DevOps Assembly Lines" style="vertical-align: middle;display: block;margin-left: auto;margin-right: auto;"/>
+
+* Selecting **Enable NAT** will enable NAT on the node pool for the subscription. All the build nodes in this node pool will use a static public IP. The static public IP can be seen after enabling NAT.
+
+<img src="/images/platform/management/add_nat_node_pool2.png" alt="Subscription NAT Node Pools with static IP for Shippable DevOps Assembly Lines" style="vertical-align: middle;display: block;margin-left: auto;margin-right: auto;"/>
+
+### Enabling NAT on existing node pool
+You can edit any node pool and enable/disable NAT by clicking on the `Edit` button the node pool header. This will take you to the **Edit Node Pool** page.
+
+<img src="/images/platform/management/edit_nat_node_pool.png" alt="Subscription Node Pools view for Shippable DevOps Assembly Lines" style="vertical-align: middle;display: block;margin-left: auto;margin-right: auto;"/>
+
+* Selecting **Enable NAT** will enable NAT on the node pool for the subscription. All the build nodes in this node pool will use a static public IP. The static public IP can be seen after enabling NAT.
+
+* While enabling/disabling the Enable NAT checkbox, a modal will appear to confirm the action
+
+<img src="/images/platform/management/enable_confirmation_nat_pool.png" alt="Subscription NAT Node Pools enable confirmation for Shippable DevOps Assembly Lines" style="vertical-align: middle;display: block;margin-left: auto;margin-right: auto;"/>
+
+<img src="/images/platform/management/disable_confirmation_nat_pool.png" alt="Subscription NAT Node Pools enable confirmation for Shippable DevOps Assembly Lines" style="vertical-align: middle;display: block;margin-left: auto;margin-right: auto;"/>
+
+### Things to Remember
+* Builds running in NAT enabled node pool will have a static IP.
+* SSH debug builds are disabled on NAT enabled node pools.
+* Whitelisting of Build Nodes is only available for paid subscriptions.
```

Comment:
We should be consistent about the terminology. I think the content should only refer to "NAT". Using "whitelisted nodes" makes it confusing.